pkgs/tools/misc/nagstamon/default.nix

```diff
@@ -1,25 +1,58 @@
-{ lib, fetchurl, pythonPackages }:
+{
+  lib,
+  fetchurl,
+  pythonPackages,
+  qt6,
+}:
 
 pythonPackages.buildPythonApplication rec {
   pname = "nagstamon";
-  version = "3.14.0";
+  version = "3.16.2";
 
   src = fetchurl {
     url = "https://github.com/HenriWahl/Nagstamon/archive/refs/tags/v${version}.tar.gz";
-    sha256 = "sha256-9RxQ/rfvoyjSUsY4tmAkBdVQqZYi3X6PBzQYFIeenzA=";
+    sha256 = "sha256-QuXs0muY0CvEJFB7ehMhvcQG4MByp3ySEmr7qB1enEk=";
   };
 
   # Test assumes darwin
   doCheck = false;
 
-  build-system = with pythonPackages; [ setuptools ];
-  dependencies = with pythonPackages; [ configparser pyqt6 psutil requests
-     beautifulsoup4 keyring requests-kerberos lxml dbus-python python-dateutil pysocks ];
+  buildInputs = [
+    qt6.qtmultimedia
+    qt6.qtbase
+    qt6.qtsvg
+  ];
+
+  # Correction : buildSystem -> nativeBuildInputs
```

Comment:
Don't rename, those attr names are what the docs use. They probably want to do something under the hood with them in the future.

Keep the wrapQtAppsHook in nativeBuildInputs

Reply:
We are only using build-system when pyproject is set to true